src/views/sources/addSourceModal.tsx

```diff
@@ -121,7 +122,7 @@ const useSourceForm = ({
         name: name,
         credentials: credentials?.map(c => Number(c)),
         hosts: hosts?.split(','),
-        port: port || (isNetwork ? '22' : '443'),
+        port: port || (isOpenshift && '6443') || (isNetwork && '22') || '443',
```

Comment:
required: looks like we're missing `isOpenshift` in the `useCallback` deps, line `139` we should mirror what we did with `isNetwork`

```
    [isNetwork, isOpenshift, formData, source, typeValue]
  );
```

future us note:
looks like the linter picked up a few other instances, i'll open a clean up PR to pick an instance in `login` and `addCreds` with the same warning

Reply:
done